### PR TITLE
[V3] Add Dict methods concept files

### PR DESCRIPTION
**Pull request by BethanyG**
Added/wrote the following files for the `dict-methods` concept:
* `about.md`
*  `introduction.md`
*  `links.json`

### CODE REVIEW

**Comment by BethanyG** [comment]
Thanks for the sharp eye, @valentin-p !  Much appreciated. 🌟

**Review by valentin-p — APPROVED** [review]

File: concepts/dict-methods/about.md [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+{'Purple baseline': '#161748',
+ 'Green Treeline': '#478559',
+ 'Factory Stone Purple': '#7c677f',
+ 'Purple Mountains Majesty': '#8076a3',
+ 'Grassy Green': '#9bc400',
+ 'Misty Mountain Pink': '#f9c5bd'} 
+
+```
+
+Swapping keys and values reliably in a dictionary takes a little more work, but can be efferently accomplished via a loop using `dict.items()`.  However, if values are not unique, extra checks are required.  Both methods assume that `dict` keys and values are _hashable_.
```
Comment (BethanyG) [inline concepts/dict-methods/about.md:223]:
```suggestion
Swapping keys and values reliably in a dictionary takes a little more work, but can be accomplished via a loop using `dict.items()`.  But if the values stored in the `dict` are not unique, extra checks are required.  Both methods assume that `dict` keys and values are _hashable_.
```

**Comment by BethanyG** [comment]
@cmccandless  - this appears to be stuck in a state where I can't re-request a review from @valentin-p, but can't approve or merge it myself.  Could you do the honors please?  **Thanks!**

**Review by cmccandless — APPROVED** [review]
Approving to unblock merge (I have not reviewed the contents of this PR myself)

**Pull request merged**